### PR TITLE
Balder/log less

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR
Better review in commit order.

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ namespace proton::matching {
 
 namespace {
 
-constexpr long SECONDS_BEFORE_ALLOWING_SOFT_TIMEOUT_FACTOR_ADJUSTMENT = 60;
+constexpr vespalib::duration SECONDS_BEFORE_ALLOWING_SOFT_TIMEOUT_FACTOR_ADJUSTMENT = 60s;
```
Comment (havardpe) [inline searchcore/src/vespa/searchcore/proton/matching/matcher.cpp:49]: SECONDS is no longer a relevant prefix

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp [hunk 2]
```diff
@@ -318,13 +318,14 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
             if (adjustedDuration < vespalib::duration::zero()) {
                 adjustedDuration = vespalib::duration::zero();
             }
-            bool allowedSoftTimeoutFactorAdjustment = (std::chrono::duration_cast<std::chrono::seconds>(my_clock::now() - _startTime).count() > SECONDS_BEFORE_ALLOWING_SOFT_TIMEOUT_FACTOR_ADJUSTMENT)
+            bool allowedSoftTimeoutFactorAdjustment = ((my_clock::now() - _startTime) > SECONDS_BEFORE_ALLOWING_SOFT_TIMEOUT_FACTOR_ADJUSTMENT)
                                                       && ! isDoomExplicit;
             if (allowedSoftTimeoutFactorAdjustment) {
                 _stats.updatesoftDoomFactor(request.getTimeout(), overtimeLimit, adjustedDuration);
             }
+            if ((_stats.softDoomed() < 10) || (_stats.softDoomed()%100 == 0))
```
Comment (havardpe) [inline searchcore/src/vespa/searchcore/proton/matching/matcher.cpp]: please use braces

**Review by havardpe — DISMISSED** [review]

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/matching/matcher.cpp]: Fixed

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp (hunk 2 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/matching/matcher.cpp]: Fixed

**Pull request merged**